R/pkg/R/DataFrame.R

```diff
@@ -254,7 +254,6 @@ setMethod("dtypes",
 #' @family DataFrame functions
 #' @rdname columns
 #' @name columns
-#' @aliases names
```

Comment:
`name` is on the same doc page as column (maybe we should change `@rdname` to names for R friendliness)

```
#' @rdname columns
 #' @name names
 setMethod("names",
```